packages/ui/src/bounty/components/BountySidebar/Periods.tsx

```diff
@@ -0,0 +1,69 @@
+import BN from 'bn.js'
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+
+import { BountyPeriod } from '@/bounty/types/Bounty'
+import { formatDuration } from '@/common/components/statistics'
+import { Stepper } from '@/common/components/Stepper'
+import { TextSmall } from '@/common/components/typography'
+import { DurationValue } from '@/common/components/typography/DurationValue'
+import { Colors } from '@/common/constants'
+
+interface PeriodStep {
+  title: 'Funding Period' | 'Working Period' | 'Judgement Period'
+  details: ReactNode
+  type: 'past' | 'active' | 'hideNumber'
```

Comment:
Is it a matter of preference or do you think enum will be more suitable here? Because to me union type is just shorter plus: https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant

Reply:
I agree. At first, I added such a comment to every union type you've used but then started to think that in this project there is no convention to use enums, so let's stay with the union types. I left this comment by mistake, so just ignore it ;-)